diagnostic.md

```diff
@@ -76,7 +76,7 @@ var groceryList = [
 
 What is the value of `groceryList[2]`?
 
-_Replace this text with your answer._
+a loaf of bread a countainer of milk a stick of butter
```

Comment:
groceryList[2] is 'a stick of butter'. what [2] means is grab the value of whatever is at index 2 of the array. And index's start at 0, so 'a stick of butter' is the 3rd item (or second index) of the array